week-1-practise/level-1/1-syntax-error-1.js

```diff
@@ -1,14 +1,16 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
+  return a + b + c;
 }
 
 var result = addNumbers(1, 3, 4);
 console.log(result); // Expected result 8 - Run the program and make sure it works
 
-
 // Answer these questions:
 // 1. How many parameters does the function take?
+// The fonction takes three parameters which are a, b, c.
 // 2. What is the function name?
-// 3. Where is the function called? with what parameters?
\ No newline at end of file
+//The function's name is addNumbers.
+// 3. Where is the function called? with what parameters?
+// The function is called outside of the function e.g. : var result - addNumbers();
```

Comment:
What about the parameters it is called with? Do you understand the question?

Reply:
Yes i understand but i forgot to answer it i will do it now.